package.json

```diff
@@ -31,7 +31,7 @@
     "@emotion/react": "11.9.3",
     "@emotion/styled": "11.9.3",
     "@leafygreen-ui/button": "16.0.0",
-    "@leafygreen-ui/card": "^7.0.0",
+    "@leafygreen-ui/card": "7.0.0",
```

Comment:
Looks like `yarn.lock` needs to be updated

Reply:
oh yarn.lock doesn't its already pinned at 7.0.0
![image](https://user-images.githubusercontent.com/4605522/186971845-b5b42484-c93e-4fa3-beb3-f2b0d4d2a59b.png)